magento/magento2
# Fix #21692 #21752 - logic in constructor of address validator and Locale Resolver check

### Description (*)

This fixes issue #21692 - error is described there

### Fixed Issues (if relevant)
1. magento/magento2#21692: Incorrect constructor of Magento\Sales\Model\Order\Address\Validator
2. magento/magento2#21752: Error while installing Magento from scratch if Locale Resolver is injected in cli command

### Manual testing scenarios (*)

1. Add custom module with Cli Command and inject `Magento\Sales\Model\Order\Address\Validator` into its constructor
2. Run Magento installation from Cli
3. Verify if installation is finished with success

### Contribution checklist (*)
